.devcontainer/devcontainer.json

```diff
@@ -89,36 +89,6 @@
             "false"
           ]
         },
-        {
-          "name": "E2E Extended AAD",
```

Comment:
How does one debug a specific test?

Reply:
You couldn't do that before either, could you? There is still a target for E2E Extended tests, there's just no separate target for AAD.